lib/rubocop/cop/bugcrowd/prefer_request_yielding_response_pattern.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bugcrowd
+      class PreferRequestYieldingResponsePattern < Cop
```

Comment:
Would be good to add the good / bad comments at the top here, and also possibly a note that even assigning the request to a `let` and calling it in the `before` is not any better than the copped examples (this is actually commonly done in crowdcontrol tests).